src/dotnet/Fable.Compiler/Transforms/MonadicTrampoline.fs

```diff
@@ -1,7 +1,7 @@
 ﻿// Source: http://fssnip.net/dK/title/Monadic-Trampoline
 module MonadicTrampoline
 
-#if !FABLE_COMPILER
+#if NETSTANDARD2_0 || !FABLE_COMPILER
```

Comment:
Is it not enough to just use `!FABLE_COMPILER`? I'm afraid `NETSTANDARD2_0` could be activated also in Fable compilation and then we will have problems with the trampoline again.

The semantics may also be clearer if we have only the `FABLE_COMPILER` symbol. I'm using the negation operator because I didn't want to put the Fable-specific code first, but we can change the order of the branches if that's better.

Reply:
@alfonsogarciacaro No, not enough, sometimes Fable is compiled with both symbols on dotnet and then we want the first branch. It's fine, this is code internal to the Fable compiler. There are only 2 cases:
- Fable compiled with dotnet, with or without FABLE_COMPILER (branch 1)
- Fable compiled with Fable, with FABLE_COMPILER, but not NETSTANDARD2_0 (branch 2)

Reply:
Ah, ok! I remember now we had a similar problem in another place. Hmm, in that case I'd prefer to leave the `NETSTANDARD2_0`. Or we can try to clarify the meaning of `FABLE_COMPILER` which should be "this is being compiled by Fable", but I guess sometimes you're using it to mean "this is compiled by dotnet and will be used by Fable". Would it be possible to use a different symbol for the latter case?

Reply:
@alfonsogarciacaro I'm afraid we already sprinkled FABLE_COMPILER all over Fable and FCS, that's how we roll.
In this particular case, I can leave only NETSTANDARD2_0 but it's safer to keep both, in case some future dotnet framework version does not define it.

Reply:
OK. I'm afraid to get the wrong code silently in some compilations if the build conditions change. But we can revisit this later, thank you!